hr_timesheet_overtime/security/ir.model.access.csv

```diff
@@ -0,0 +1,6 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_resource_overtime,resource.overtime,model_resource_overtime,base.group_system,1,1,1,1
+access_resource_overtime_rate,resource.overtime.rate,model_resource_overtime_rate,base.group_system,1,1,1,1

```

Comment:
are you sure that this should only go for the admin user ? I would rather choose the HR manager for this.

Reply:
admin user right limitation should only be set at for system wide purpose. Or for deletation only. Here you'll need to give super power to create and update a resource overtime rate.